VORP-Housing[Client-Server]/vorphousing_sv/vorp_housing_sv_init.cs

```diff
@@ -21,107 +20,134 @@ public vorp_housing_sv_init()
+                dynamic result = await Exports["ghmattimysql"].executeSync("SELECT * FROM housing", new string[] { });
 
-                    Dictionary<uint, House> _Houses = Houses.ToDictionary(h => h.Key, h => h.Value);
+                if (result != null && result.Count > 0)
+                {
+                    Logger.Debug(JsonConvert.SerializeObject(result));
 
-                    if (result.Count != 0)
+                    foreach (var r in result)
                     {
-                        foreach (var r in result)
+                        uint houseId = ConvertValue(r.id.ToString());
+                        string identifier = r.identifier;
+                        int charidentifier = r.charidentifier;
+                        string furniture = "{}";
+                        if (!String.IsNullOrEmpty(r.furniture))
                         {
-                            uint houseId = ConvertValue(r.id.ToString());
-                            string identifier = r.identifier;
-                            int charidentifier = r.charidentifier;
-                            string furniture = "{}";
-                            if (!String.IsNullOrEmpty(r.furniture))
-                            {
-                                furniture = r.furniture;
-                            }
-                            _Houses[houseId].Identifier = identifier;
-                            _Houses[houseId].CharIdentifier = charidentifier;
-                            _Houses[houseId].Furniture = furniture;
-                            _Houses[houseId].IsOpen = Convert.ToBoolean(r.open);
-
-                            if (identifier.Equals(sid))
-                            {
-                                _Houses[houseId].IsOwner = true;
-                            }
+                            furniture = r.furniture;
+                        }
+                        _Houses[houseId].Identifier = identifier;
+                        _Houses[houseId].CharIdentifier = charidentifier;
+                        _Houses[houseId].Furniture = furniture;
+                        _Houses[houseId].IsOpen = Convert.ToBoolean(r.open);
 
+                        if (identifier.Equals(sid))
+                        {
+                            _Houses[houseId].IsOwner = true;
                         }
-                        string houses = JsonConvert.SerializeObject(_Houses);
-                        cb(houses);
-                    }
-                    else
-                    {
-                        string houses = JsonConvert.SerializeObject(_Houses);
-                        cb(houses);
+
                     }
+                    string houses = JsonConvert.SerializeObject(_Houses);
+                    TriggerClientEvent("vorp_housing:ListHouses", houses);
+                    //cb(houses);
                 }
-                catch (Exception e)
+                else
```

Comment:
Same question as before with the `_Rooms`, I do not believe the else is required.